### PR TITLE
Added otel metrics collector

### PR DESCRIPTION
**Pull request by xinpan-tibco**

### CODE REVIEW

**Review by syan-tibco — APPROVED** [review]
After adding enabled flag you can push the changes

File: charts/dp-otel-collectors/templates/metrics-collector.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: {{ .Values.metrics-collector.name }}
```
Comment (efloresb-tibco) [inline charts/dp-otel-collectors/templates/metrics-collector.yaml:4]: The collector's name might not be enough. It needs to be unique across cluster. Maybe we need to add the release name here as well.

Comment (xinpan-tibco) [reply charts/dp-otel-collectors/templates/metrics-collector.yaml:4]: I see, so it needs a release name as prefix, such as {{ .Release.Name }}-{{ .Values.metrics-collector.name }}. 
Let me update.

**Comment by syan-tibco** [comment]
@xinpan-tibco  please also update chart version. So that we can have a new release of this chart. 

change https://github.com/syan-tibco/tp-helm-charts/blob/main/charts/dp-otel-collectors/Chart.yaml#L3 to 1.0.2

cc: @efloresb-tibco

**Pull request merged**